src/rhsm/profile.py

```diff
@@ -75,7 +75,14 @@ def _uniquify(module_list):
         ret = {}
         for module in module_list:
             key = (module["name"], module["stream"], module["version"], module["context"], module["arch"])
-            ret[key] = module
+            # Prefer duplicates that are Active.
+            # There are "enabled" duplicates from dnf that are marked as inactive.
+            if ret.get(key) is not None:
+                if not ret[key]["active"]:
```

Comment:
How sure are we that "active" will always be available?
Should we surround this with a try..except ? Or perhaps we should use 'get' again with a default of False.